### ISSUE TITLE
Complete API Review and Approval Process for v1alpha2

### ISSUE THREAD

**Issue by robscott** [kubernetes-sigs/gateway-api#716]
As we're transitioning to a k8s.io API Group, we'll need formal review and approval for each new API release. Once this is complete, we'll need to [update the annotations on the CRDs](https://github.com/kubernetes-sigs/gateway-api/blob/master/hack/update-codegen.sh#L56) to reference the formal approval.

**Comment by hbagdi** [comment]
> we'll need formal review and approval for each new API release

Are there any guidelines written up for this? Maybe not since there aren't many ~(any?)~ APIs outside k/k that are governed by the same approval process. 
Do we have to follow the process for alpha APIs? What about beta?

**Comment by hbagdi** [comment]
I saw the v1alpha2 milestone after I posted my comment above. That explains that we need a review from API reviewers before any new API release.

**Comment by robscott** [comment]
I think [this is the best documentation](https://github.com/kubernetes/community/blob/master/sig-architecture/api-review-process.md) around the process. I need to figure out exactly which parts of that we need to go through. I know in the past when I'd asked about a KEP for Gateway API it was considered unnecessary, but maybe it's required for this part of the process. Also want to check in with @thockin to see if he's more familiar with this process.

**Comment by bowei** [comment]
The KEP should be pretty skeletal (polint to our website)

Bowei

**Comment by liggitt** [comment]
+1 to getting @thockin ack on the k8s.io change as soon as possible

**Comment by youngnick** [comment]
Okay, so kubernetes/enhancements#2830 has merged, and I think we are down to one last feedback item from #861 (the backup review PR). Tim requested that we clarify if all Routes *must* have a `ParentRef` fields - I think the answer is yes, but I'll log a separate issue for it.

I *believe* that once we've addressed that final feedback, the API review process is done, so we should be good to close this out and mark the review as finished? @robscott, thoughts?

**Comment by robscott** [comment]
@andrewsykim @danwinship @khenidak @thockin Do you have any remaining feedback that needs to be addressed before we release v1alpha2?

**Comment by robscott** [comment]
Missed closing this earlier, v1alpha2 was formally approved as part of the corresponding changelog PR (https://github.com/kubernetes-sigs/gateway-api/pull/891).

/close
